server/data/strapiApiClient.test.ts

```diff
@@ -163,7 +163,7 @@ describe('strapiApiClient', () => {
     describe('getComponent', () => {
       it('should return a single component', async () => {
         const component = {
-          data: [{ id: 1, attributes: { name: 'component' } }],
+          data: { id: 1, attributes: { name: 'component' } },
```

Comment:
DataItem currently has this as a single value - it could be a list but the types might get super complex. The wrapper handles this nicely so it doesn't really matter